modular_skyrat/modules/altborgs/code/tallborg/robot.dm

```diff
@@ -18,6 +18,53 @@
 	else
 		regenerate_icons()
 
+// Brand stamps for cyborgs
+/mob/living/silicon/robot/examine(mob/user)
+	. = ..()
+
+	var/mob/living/silicon/robot/robot = src
+	if(R_TRAIT_TALL in robot.model.model_features)
+		. += span_notice("<i>It seems to have branding insignia, you can examine again to take a closer look...")
+	else
+		return
+
+/mob/living/silicon/robot/examine_more(mob/user)
+	. = ..()
+
+	var/mob/living/silicon/robot/robot = src
+	. += span_notice("<i>You examine [src] closer, and note the following...</i>")
+
+	switch(user.icon)
+		if(CYBORG_ICON_PEACEKEEPER_TALL)
+			. += "Its waist has <i>[span_pink("Zeng-Hu Pharmaceuticals")]</i> stamped onto it."
+		if(CYBORG_ICON_MINING_TALL)
+			. += "Its waist has <i>[span_cyan("Interdyne Pharmaceuticals")]</i> laser-cut onto it."
```

Comment:
Not sure about this one, sounds a bit counter-intuitive that they'd get cyborg models, of all things, from Syndicate contractors.

Reply:
Well, technically, they arent _strictly_ syndicate, and they already can have dart weapons sold by them via guncargo.
But, yeah, having CYBORG models sold by them feels off, especially considering that NT is supposed to be dominant in the AI/Cyborg industry IIRC